Resources/Private/JavaScript/asset-usage/src/components/AssetUsagesToggleButton.module.css

```diff
@@ -0,0 +1,3 @@
+.assetUsageBadge {
+    color: var(--theme-blue);
```

Comment:
I don't think the blue on white for the badge fits well with the rest of the module.
I would rather use white on one of our grey colors.